tarkalabs/tarka-ui-kit-android
# Crash Fix: when switching tabs in WoDetail Activity.

### What was Implemented/Fixed 📚
<img width="1225" alt="Screenshot 2024-04-15 at 5 01 21 PM" src="https://github.com/tarkalabs/tarka-ui-kit-android/assets/99186954/197ec011-7be1-47d2-b124-b871600ed73f">

- activity reference got properly inside compose function.

### Links to tickets/documentation 🔗
- [https://eam360.atlassian.net/browse/AMA-7112](https://eam360.atlassian.net/browse/AMA-7112)

### Current Issues ❌
We need use the updated TarkaUiKit in Technician App to get out of crash mentioned in jira card.

### How it Was Implemented 🔦
[new Extension function added to get Activity properly](https://github.com/tarkalabs/tarka-ui-kit-android/commit/75e56caa84c4be6d780cd3132f29b41738b7b64e)

[parent activity accessed properly from compose to get a window.](https://github.com/tarkalabs/tarka-ui-kit-android/commit/b9225357baa169b8af5fc8b0f3d8064c4cd0dd60)

### How can a reviewer test this change 🧑‍💻
- Run this Branch and check any compose with the Updated Theme wrapping i.e no crash should exist.

### Testing checks / How Has This Been Tested ✅
